web/js/layers/layers.info.js

```diff
@@ -1,7 +1,7 @@
-var wv = wv || {};
-wv.layers = wv.layers || {};
+import $ from 'jquery';
+import wvui from '../ui/ui';
 
-wv.layers.info = wv.layers.info || function (config, models, layer) {
+export function layersInfo(config, models, layer) {
```

Comment:
What do you think about keeping the name `info`, importing this into the `layers` parent, and attaching it to the object that exports?